tools/create_yocto_rz_src.sh

```diff
@@ -180,26 +202,61 @@ function unpack_multi_os(){
 	local zip_dir="REN_MULTI_OS"
 	local rtos="meta-rz*.tar.gz"
 
-	extract_to_meta ${pkg_file} ${zip_dir} ${rtos} ${YOCTO_HOME}
+	extract_to_meta "${pkg_file}" "${zip_dir}" "${rtos}" "${YOCTO_HOME}"
 	rm -fr ${zip_dir}
 }
 
 function remove_redundant_patches(){
 	# remove linux patches that were merged into the Avnet kernel
-	flist=$(find ${YOCTO_HOME} -name "linux-renesas_*.bbappend")
+	flist=$(find "${YOCTO_HOME}" -name "linux-renesas_*.bbappend")
 	for ff in ${flist}
 	do
-		echo ${ff}
-		rm -rf ${ff}
+		echo "${ff}"
+		rm -rf "${ff}"
 	done
 
 	# remove u-boot patches
-	find ${YOCTO_HOME} -name "u-boot_*.bbappend" -print -exec rm -rf {} \;
+	find "${YOCTO_HOME}" -name "u-boot_*.bbappend" -print -exec rm -rf {} \;
 
 	# remove tfa patches
-	find ${YOCTO_HOME} -name "trusted-firmware-a.bbappend" -print -exec mv {} {}.remove \;
+	find "${YOCTO_HOME}" -name "trusted-firmware-a.bbappend" -print -exec mv {} {}.remove \;
+}
+
+function clone_meta_rzboard(){
+	log_info "Cloning meta-rzboard layer"
+	git clone https://github.com/Avnet/meta-rzboard.git -b rzboard_dunfell_5.10.201
```

Comment:
Update this line from `git clone https://github.com/Avnet/meta-rzboard.git -b rzboard_dunfell_5.10.201` to `git clone https://github.com/Avnet/meta-rzboard.git -b feature/node_18_support` for verification. Once the changes are merged, it will be correct.